File: A's/py/1030A.py
def main() -> None:
    n: int = int(input())
    arr = list(map(int, input().split()))

    res: str = "EASY"
    for a in arr:
        if a == 1: res = "HARD"

    print(res)

if __name__ == "__main__":
    main()

File: A's/cpp/1030A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int n; cin >> n;
    string res = "EASY";

    while (n--) {
        int i; cin >> i;
        if (i == 1) { res = "HARD"; }
    }

    cout << res << endl;
    
    return 0;
}